thirdparty/CMakeLists.txt

```diff
@@ -150,10 +151,11 @@ if(NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_IMPORT_ENABLE)
     set(SYSTEM_PROTOC ${SYSTEM_PROTOC} PARENT_SCOPE)
     set(PROTOC_EXECUTABLE ${PROTOC_EXECUTABLE} PARENT_SCOPE)
     set(Protobuf_LIBRARIES ${Protobuf_LIBRARIES} PARENT_SCOPE)
+    set(Protobuf_LITE_LIBRARIES ${Protobuf_LITE_LIBRARIES} PARENT_SCOPE)
     set(Protobuf_INCLUDE_DIRS ${Protobuf_INCLUDE_DIRS} PARENT_SCOPE)
 
     # set public / interface compile options
-    foreach(target IN LISTS Protobuf_LIBRARIES)
+    foreach(target IN LISTS Protobuf_LITE_LIBRARIES Protobuf_LIBRARIES)
```

Comment:
If `Protobuf_LIBRARIES==Protobuf_LITE_LIBRARIES` - loop will be executed twice for same target. Is it ok?

Reply:
yes, it will functionally work. 

In we talk about performance in cmake - we don't care about it 😄